benchmarks/dc_eis_latency/dceisLatency.json

```diff
@@ -3,7 +3,7 @@
  "name": "node-dc-eis Latency[lower is better]" ,
  "template": "../common/common_template1.html.base",
  "units": "ms",
- "outputBase" : "dceis_latency",
```

Comment:
This change would change the name of the charts, and the names of the urls on benchmarking.nodejs.org.  I'd suggest removing and if we do want to make this change then adding in a PR were we also update the main page that shows the charts.

Reply:
@mhdawson Existing graphs on the main page have file names as above,
https://benchmarking.nodejs.org/charts/dceis_throughput.png, and
https://benchmarking.nodejs.org/charts/dceis_latency.png

It's not clear to me what needs to be changed?

Reply:
The PR changes the name from dceis_... to dc-eis_... adding the -

Reply:
which would break the references in https://github.com/nodejs/benchmarking/blob/master/www/index.html